drivers/hmis/app/graphql/types/hmis_schema/enums/hud.rb

```diff
@@ -957,13 +968,13 @@ class SSVFFinancialAssistance < Types::BaseEnum
     value 'TRANSPORTATION_SERVICES_TOKENS_VOUCHERS', '(8) Transportation services: tokens/vouchers', value: 8
     value 'TRANSPORTATION_SERVICES_VEHICLE_REPAIR_MAINTENANCE', '(9) Transportation services: vehicle repair/maintenance', value: 9
     value 'CHILD_CARE', '(10) Child care', value: 10
-    value 'GENERAL_HOUSING_STABILITY_ASSISTANCE_EMERGENCY_SUPPLIES_DEPRECATED', '(11) General housing stability assistance - emergency supplies [Deprecated]', value: 11
     value 'GENERAL_HOUSING_STABILITY_ASSISTANCE', '(12) General housing stability assistance', value: 12
     value 'EMERGENCY_HOUSING_ASSISTANCE', '(14) Emergency housing assistance', value: 14
     value 'SHALLOW_SUBSIDY_FINANCIAL_ASSISTANCE', '(15) Shallow Subsidy - Financial Assistance', value: 15
     value 'FOOD_ASSISTANCE', '(16) Food Assistance', value: 16
     value 'LANDLORD_INCENTIVE', '(17) Landlord Incentive', value: 17
     value 'TENANT_INCENTIVE', '(18) Tenant Incentive', value: 18
+    value 'GENERAL_HOUSING_STABILITY_ASSISTANCE_EMERGENCY_SUPPLIES_DEPRECATED', '(11) General housing stability assistance - emergency supplies [Deprecated]', value: 11
```

Comment:
This was deprecated in 2024 (or before), I think we can drop it entirely from the enum